src/util/homePageErrors.js

```diff
@@ -0,0 +1,9 @@
+const homePageErrors = {
+  firstName: "Not recognized as a proper name",
+  lastName: "Not recognized as a proper name",
```

Comment:
For a more user friendly experience, you could add more information to these errors - i.e. telling the user what characters are allowed?